### PR TITLE
chore(website): a small redesign

### PR DESCRIPTION
**Pull request by egoist**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [x] Other, please describe:

If changing the UI/CSS related code, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by MarchWorks** [comment]
I like it 👍

File: website/src/components/Header.vue [hunk 1]
```diff
@@ -78,19 +77,12 @@
...
   methods: {
-    toggleSidebar() {
-      if (document.body.classList.contains('show-sidebar')) {
-        document.body.classList.remove('show-sidebar')
+    toggleLeftbar() {
+      if (document.body.classList.contains('show-leftbar')) {
+        document.body.classList.remove('show-leftbar')
       } else {
-        document.body.classList.add('show-sidebar')
+        document.body.classList.add('show-leftbar')
```
Comment (kidonng) [inline website/src/components/Header.vue]: Why not `document.body.classList.toggle('show-leftbar')`

Comment (egoist) [reply website/src/components/Header.vue]: Oh I didn't know about that 😂

**Comment by kidonng** [comment]
Content overflow (on Firefox)

**Pull request merged**